File: fix_es_faq1_newline.py
#!/usr/bin/env python3
import re

file_path = './src/locales/translations.ts'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Replace \\\\n\\\\n with \\n\\n in Spanish FAQ1Answer
pattern = r'("ChefNet es una plataforma global de FoodTech que unifica el descubrimiento, reservas, pagos y engagement con restaurantes y chefs privados\.)\\\\\\\\n\\\\\\\\n(Aborda la fragmentación del ecosistema de servicios de alimentación mediante el establecimiento de un único estándar integrado y sin fricciones\.",)'
replacement = r'\1\\n\\n\2'

content = re.sub(pattern, replacement, content)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print('✅ Fixed Spanish FAQ1 newlines: \\\\\\\\n\\\\\\\\n → \\n\\n')

File: fix_es_faq1_newline.js
const fs = require('fs');

const filePath = './src/locales/translations.ts';
let content = fs.readFileSync(filePath, 'utf8');

// Replace the four backslashes with two backslashes in Spanish FAQ1Answer
content = content.replace(
  /("ChefNet es una plataforma global de FoodTech que unifica el descubrimiento, reservas, pagos y engagement con restaurantes y chefs privados\.)(\\\\\\\\n\\\\\\\\n)(Aborda la fragmentación del ecosistema de servicios de alimentación mediante el establecimiento de un único estándar integrado y sin fricciones\.",)/,
  '$1\\n\\n$3'
);

fs.writeFileSync(filePath, content, 'utf8');
console.log('✅ Fixed Spanish FAQ1 newlines: \\\\\\\\n\\\\\\\\n → \\n\\n');
